tests/accelerators/test_accelerator_connector.py

```diff
@@ -174,13 +179,15 @@ def on_fit_start(self, trainer, pl_module):
             raise SystemExit()
 
     model = BoringModel()
-    trainer = Trainer(fast_dev_run=True, accelerator="ddp", gpus=2, callbacks=[CB()])
+    with pytest.deprecated_call(match=r"accelerator='ddp'\)` has been deprecated in v1.5"):
```

Comment:
why are we adding these deprecation tests in some places and update to the new api in other places? 
Shouldn't we set strategy=ddp here?

Reply:
if I'm not mistaken, @kaushikb11 duplicated most of the `accelerator` flag tests but using the `strategy` flag.

This is one of them.